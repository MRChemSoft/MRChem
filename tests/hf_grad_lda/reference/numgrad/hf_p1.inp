world_prec = 1.0e-6                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n
world_unit = angstrom

# use Psi4 naming of displacements
# displacement 4 of 5 (i.e. point +1 in the stencil) 
Molecule {
$coords
F           -1.194797389853    -0.244603028004     0.340822581459
H           -1.673845087234    -0.395504779624     1.109456843865
$end
translate = false
}

WaveFunction {
  method = LDA                      # Wave function method (HF or DFT)
}

SCF {
  guess_type = CORE_DZ             # Type of initial guess: none, mw, gto
}
