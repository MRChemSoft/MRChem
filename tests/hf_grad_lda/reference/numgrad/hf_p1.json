{
  "input": {
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "f",
          "xyz": [
            -2.2578398412398952,
            -0.46223273218172767,
            0.644061336045467
          ]
        },
        {
          "atom": "h",
          "xyz": [
            -3.163108789922591,
            -0.7473957144698308,
            2.096569581996545
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -6
    },
    "printer": {
      "file_name": "hf_p1.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": false,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "svwn5"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (SVWN5)",
        "prec": 0.001,
        "restricted": true,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "energy_thrs": -1.0,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 5,
        "localize": false,
        "max_iter": 100,
        "method": "DFT (SVWN5)",
        "orbital_thrs": 9.999999999999999e-06,
        "rotation": 0,
        "start_prec": 1e-06
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -2.303448215294742,
        -0.4765995344251016,
        0.7172401847130201
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.7070132120618791,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -0.3688734818704127,
            -0.11619666399600437,
            0.5918600946837467
          ],
          "vector_el": [
            23.114793879160988,
            4.7912936401537,
            -7.30126151172811
          ],
          "vector_nuc": [
            -23.4836673610314,
            -4.907490304149705,
            7.893121606411857
          ]
        }
      },
      "geometric_derivative": {
        "electronic": [
          0.0,
          0.0,
          0.0
        ],
        "electronic_norm": 0.0,
        "nuclear": [
          0.0,
          0.0,
          0.0
        ],
        "nuclear_norm": 0.0,
        "total": [
          0.0,
          0.0,
          0.0
        ],
        "total_norm": 0.0
      },
      "geometry": [
        {
          "symbol": "F",
          "xyz": [
            -2.2578398412398952,
            -0.46223273218172767,
            0.644061336045467
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -3.163108789922591,
            -0.7473957144698308,
            2.096569581996545
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -24.087668763752912,
          -1.096705034014673,
          -0.49879440257194513,
          -0.3608004534005204,
          -0.3608004258297083
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": -52.80953815913952
      },
      "scf_energy": {
        "E_ee": 55.4561584053771,
        "E_eext": 0.0,
        "E_el": -105.03897347768539,
        "E_en": -249.7765131598092,
        "E_kin": 99.42010871778531,
        "E_next": 0.0,
        "E_nn": 5.186988193730312,
        "E_nuc": 5.186988193730312,
        "E_tot": -99.85198528395507,
        "E_x": 0.0,
        "E_xc": -10.138727441038572
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 20,
      "routine": "mrchem.x",
      "total_cores": 20,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 94.5899270415703,
        "E_eext": 0.0,
        "E_el": -83.2211012756751,
        "E_en": -329.2562920287282,
        "E_kin": 166.7526174520884,
        "E_next": 0.0,
        "E_nn": 5.186988193730312,
        "E_nuc": 5.186988193730312,
        "E_tot": -78.03411308194478,
        "E_x": 0.0,
        "E_xc": -15.307353740605627
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 58.07815071602484,
              "E_eext": 0.0,
              "E_el": -104.41672676582951,
              "E_en": -247.21781083918046,
              "E_kin": 95.02111386424895,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.2297385720992,
              "E_x": 0.0,
              "E_xc": -10.298180506922828
            },
            "energy_total": -99.2297385720992,
            "energy_update": 21.195625490154413,
            "mo_residual": 1.1964202835378728,
            "wall_time": 26.078034647
          },
          {
            "energy_terms": {
              "E_ee": 50.759769405144645,
              "E_eext": 0.0,
              "E_el": -104.72560189190384,
              "E_en": -239.3325757475304,
              "E_kin": 93.34898244261707,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.53861369817352,
              "E_x": 0.0,
              "E_xc": -9.501777992135178
            },
            "energy_total": -99.53861369817352,
            "energy_update": 0.30887512607432654,
            "mo_residual": 0.5033765630755308,
            "wall_time": 22.793071573
          },
          {
            "energy_terms": {
              "E_ee": 57.25764444261795,
              "E_eext": 0.0,
              "E_el": -104.98269248839145,
              "E_en": -251.96735870134313,
              "E_kin": 100.06312302273047,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.79570429466114,
              "E_x": 0.0,
              "E_xc": -10.336101252396729
            },
            "energy_total": -99.79570429466114,
            "energy_update": 0.2570905964876147,
            "mo_residual": 0.3662551813193571,
            "wall_time": 25.983321465
          },
          {
            "energy_terms": {
              "E_ee": 54.92476599213474,
              "E_eext": 0.0,
              "E_el": -105.02352851478851,
              "E_en": -248.33059073720665,
              "E_kin": 98.4377743901561,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.8365403210582,
              "E_x": 0.0,
              "E_xc": -10.0554781598727
            },
            "energy_total": -99.8365403210582,
            "energy_update": 0.0408360263970593,
            "mo_residual": 0.1281971145381905,
            "wall_time": 23.181131337
          },
          {
            "energy_terms": {
              "E_ee": 55.762317538806194,
              "E_eext": 0.0,
              "E_el": -105.0367380116125,
              "E_en": -250.3694529336273,
              "E_kin": 99.74599932241831,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.84974981788218,
              "E_x": 0.0,
              "E_xc": -10.175601939209743
            },
            "energy_total": -99.84974981788218,
            "energy_update": 0.013209496823989753,
            "mo_residual": 0.07898510897855178,
            "wall_time": 24.920219549
          },
          {
            "energy_terms": {
              "E_ee": 55.42965971924536,
              "E_eext": 0.0,
              "E_el": -105.03867844024171,
              "E_en": -249.74902591356346,
              "E_kin": 99.41546342243274,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.8516902465114,
              "E_x": 0.0,
              "E_xc": -10.134775668356356
            },
            "energy_total": -99.8516902465114,
            "energy_update": 0.0019404286292115103,
            "mo_residual": 0.022206343780512203,
            "wall_time": 24.307760371
          },
          {
            "energy_terms": {
              "E_ee": 55.45362402475744,
              "E_eext": 0.0,
              "E_el": -105.03895597358688,
              "E_en": -249.78471851253474,
              "E_kin": 99.4307751158359,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.85196777985657,
              "E_x": 0.0,
              "E_xc": -10.13863660164547
            },
            "energy_total": -99.85196777985657,
            "energy_update": 0.00027753334516944506,
            "mo_residual": 0.009678269613832833,
            "wall_time": 25.052642371
          },
          {
            "energy_terms": {
              "E_ee": 55.45802446748698,
              "E_eext": 0.0,
              "E_el": -105.038970385171,
              "E_en": -249.78258762795315,
              "E_kin": 99.42459911723893,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.85198219144068,
              "E_x": 0.0,
              "E_xc": -10.13900634194375
            },
            "energy_total": -99.85198219144068,
            "energy_update": 1.441158411807919e-05,
            "mo_residual": 0.0012025594644904272,
            "wall_time": 24.684418769
          },
          {
            "energy_terms": {
              "E_ee": 55.456278602399216,
              "E_eext": 0.0,
              "E_el": -105.03897322052012,
              "E_en": -249.7743341222979,
              "E_kin": 99.41778813881916,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.8519850267898,
              "E_x": 0.0,
              "E_xc": -10.1387058394406
            },
            "energy_total": -99.8519850267898,
            "energy_update": 2.835349121710351e-06,
            "mo_residual": 0.0007257887830793323,
            "wall_time": 23.678274391
          },
          {
            "energy_terms": {
              "E_ee": 55.455966742470125,
              "E_eext": 0.0,
              "E_el": -105.03897340870199,
              "E_en": -249.77479802988745,
              "E_kin": 99.41854034274478,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.85198521497168,
              "E_x": 0.0,
              "E_xc": -10.13868246402945
            },
            "energy_total": -99.85198521497168,
            "energy_update": 1.881818718629802e-07,
            "mo_residual": 0.00017973851688879414,
            "wall_time": 24.115345378
          },
          {
            "energy_terms": {
              "E_ee": 55.456212242458086,
              "E_eext": 0.0,
              "E_el": -105.03897347230536,
              "E_en": -249.77682383761447,
              "E_kin": 99.42037894553341,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.85198527857504,
              "E_x": 0.0,
              "E_xc": -10.138740822682385
            },
            "energy_total": -99.85198527857504,
            "energy_update": 6.360336612942774e-08,
            "mo_residual": 9.912942043909034e-05,
            "wall_time": 23.427944578
          },
          {
            "energy_terms": {
              "E_ee": 55.45612822845301,
              "E_eext": 0.0,
              "E_el": -105.03897347638369,
              "E_en": -249.77670319558433,
              "E_kin": 99.4203312219545,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.85198528265337,
              "E_x": 0.0,
              "E_xc": -10.13872973120685
            },
            "energy_total": -99.85198528265337,
            "energy_update": 4.0783305621516774e-09,
            "mo_residual": 2.1738314965744726e-05,
            "wall_time": 24.774097034
          },
          {
            "energy_terms": {
              "E_ee": 55.456205824536376,
              "E_eext": 0.0,
              "E_el": -105.0389734773317,
              "E_en": -249.77671344074156,
              "E_kin": 99.42027023399213,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.85198528360138,
              "E_x": 0.0,
              "E_xc": -10.138736095118652
            },
            "energy_total": -99.85198528360138,
            "energy_update": 9.480061180511257e-10,
            "mo_residual": 1.734279634080589e-05,
            "wall_time": 23.766770945
          },
          {
            "energy_terms": {
              "E_ee": 55.4561584053771,
              "E_eext": 0.0,
              "E_el": -105.03897347768539,
              "E_en": -249.7765131598092,
              "E_kin": 99.42010871778531,
              "E_next": 0.0,
              "E_nn": 5.186988193730312,
              "E_nuc": 5.186988193730312,
              "E_tot": -99.85198528395507,
              "E_x": 0.0,
              "E_xc": -10.138727441038572
            },
            "energy_total": -99.85198528395507,
            "energy_update": 3.5369396300666267e-10,
            "mo_residual": 9.26666425296015e-06,
            "wall_time": 24.811483964
          }
        ],
        "wall_time": 341.576153203
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
