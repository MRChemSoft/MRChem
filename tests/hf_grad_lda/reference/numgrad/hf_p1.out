                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         mrchem-gradient                          ***
***         Git commit hash    c6b193dd4223e5895077                     ***
***         Git commit author  Roberto Di Remigio                       ***
***         Git commit date    Fri Sep 25 22:32:15 2020 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                              20
 Total cores             :                                              20
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.3.1
 Git branch            : mrchem-gradient
 Git commit hash       : b79bdf06c3c82a53edb6
 Git commit author     : Roberto Di Remigio
 Git commit date       : Tue Sep 15 18:31:50 2020 +0200

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : MPI/OpenMP

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   64.00000    64.00000    64.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -64.00000   -64.00000   -64.00000 ]
 upper bounds          : [   64.00000    64.00000    64.00000 ]
 total length          : [  128.00000   128.00000   128.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257840       -0.462233        0.644061
    1       H            :       -3.163109       -0.747396        2.096570
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : True
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999932360553e-01
    1    2    p          :                              9.999511478174e-01
    2    2    p          :                              9.999999998857e-01
    3    2    p          :                              9.999999964893e-01
    4    2    p          :                              9.999999873497e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (SVWN5)
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                166.752617452088
 E-N energy              :            (au)               -329.256292028728
 Coulomb energy          :            (au)                 94.589927041570
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.307353740606
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.186988193730
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -83.221101275675
 Nuclear energy          :            (au)                  5.186988193730
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.803411308194e+01
                         :      (kcal/mol)             -4.896714486394e+04
                         :        (kJ/mol)             -2.048785342437e+05
                         :            (eV)             -2.123416298108e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -14.603338334738
    1    2    p          :            (au)                  3.227532004122
    2    2    p          :            (au)                  4.749779633192
    3    2    p          :            (au)                  4.775150966130
    4    2    p          :            (au)                  5.080830528635
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  6.459909594683
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (SVWN5)
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          2.236068e+00         -78.034113081945         -7.803411e+01
    1          1.196420e+00         -99.229738572099         -2.119563e+01
    2          5.033766e-01         -99.538613698174         -3.088751e-01
    3          3.662552e-01         -99.795704294661         -2.570906e-01
    4          1.281971e-01         -99.836540321058         -4.083603e-02
    5          7.898511e-02         -99.849749817882         -1.320950e-02
    6          2.220634e-02         -99.851690246511         -1.940429e-03
    7          9.678270e-03         -99.851967779857         -2.775333e-04
    8          1.202559e-03         -99.851982191441         -1.441158e-05
    9          7.257888e-04         -99.851985026790         -2.835349e-06
   10          1.797385e-04         -99.851985214972         -1.881819e-07
   11          9.912942e-05         -99.851985278575         -6.360337e-08
   12          2.173831e-05         -99.851985282653         -4.078331e-09
   13          1.734280e-05         -99.851985283601         -9.480061e-10
   14          9.266664e-06         -99.851985283955         -3.536940e-10
---------------------------------------------------------------------------
                      SCF converged in 14 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257840       -0.462233        0.644061
    1       H            :       -3.163109       -0.747396        2.096570
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 99.420108717785
 E-N energy              :            (au)               -249.776513159809
 Coulomb energy          :            (au)                 55.456158405377
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -10.138727441039
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.186988193730
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -105.038973477685
 Nuclear energy          :            (au)                  5.186988193730
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.985198528396e+01
                         :      (kcal/mol)             -6.265806626413e+04
                         :        (kJ/mol)             -2.621613494193e+05
                         :            (eV)             -2.717110819569e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -24.087668763753
    1    2    p          :            (au)                 -1.096705034015
    2    2    p          :            (au)                 -0.498794402572
    3    2    p          :            (au)                 -0.360800453401
    4    2    p          :            (au)                 -0.360800425830
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -52.809538159140
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       23.114794        4.791294       -7.301262
 Magnitude               :            (au)                       24.709484
                         :         (Debye)                       62.805238
---------------------------------------------------------------------------
 Nuclear vector          :      -23.483667       -4.907490        7.893122
 Magnitude               :            (au)                       25.256038
                         :         (Debye)                       64.194440
---------------------------------------------------------------------------
 Total vector            :       -0.368873       -0.116197        0.591860
 Magnitude               :            (au)                        0.707013
                         :         (Debye)                        1.797048
===========================================================================


===========================================================================
                           Geometric Derivative
---------------------------------------------------------------------------
 Total                   :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
---------------------------------------------------------------------------
 Nuclear                 :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
---------------------------------------------------------------------------
 Electronic              :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  5m 48s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
